src/engine/HedronEngine.ts

```diff
@@ -79,11 +79,12 @@ export class HedronEngine {
 
   run() {
     const debugScene = createDebugScene(this.renderer.aspectRatio)
+    debugScene.renderer = this.renderer.composer?.getRenderer()
```

Comment:
Pprobably want to refactor it to `createDebugScene(this.renderer)`. It just was so many layers in deep and I stayed up way too late getting this feature/my sketch working. Will fix before merge